packages/core/src/components/spinner/spinner.scss

```diff
@@ -0,0 +1,70 @@
+@use '../../animations';
+@use '../../helpers';
+@use '../../mixins';
+@use '../includes';
+
+@mixin Spinner() {
+  @if not includes.the('spin-animation') {
+    @include animations.Spin();
+  }
+
+  .ods-spinner {
+    animation: ods-spin 0.6s linear infinite;
+    border: 0 solid helpers.color('background-spinner');
+    border-left-color: helpers.color('content-spinner');
+    border-radius: helpers.border-radius('full');
+
+    &.-large {
+      @include mixins.square(helpers.space(6));
+
+      border-width: helpers.space(0.5);
+    }
+
+    &.-medium {
+      @include mixins.square(helpers.space(3));
+
+      border-width: helpers.space(0.375);
+    }
+
+    &.-small {
+      @include mixins.square(helpers.space(2));
+
+      border-width: helpers.space(0.25);
+    }
+  }
+
+  .ods-spinner-container {
```

Comment:
I don't think the `container` or `label` are needed.

Wouldn't this work?

```scss
.ods-spinner { // as container
  color: secondary; // will only affect text anyway
  display: grid;
  // ... all other necessary positioning styles

  &.-large {
    gap: space(0.5); // replaces margin-top/left
  }

  &:before { // as spinner
    @include animation.Spin();
  }  
}
```

Reply:
You're right, I managed to get rid of all of it!